# Curso em Vídeo - Exercícios de Python 3 - Mundo 1

## Aulas 1, 2, 3 e 4

### Exercício 001: Crie um programa que escreva "Olá, mundo!" na tela.

In [1]:
print("Olá, Mundo!")

Olá, Mundo!


### Exercício 002: Faça um programa que leia o nome de uma pessoa e mostre uma mensagem de boas-vindas.

In [2]:
nome = input("Digite seu nome: ")
print(f"Seja bem vindo, {nome}")

Digite seu nome: Deryck
Seja bem vindo, Deryck


## Aula 6

### Exercício 003: Crie um programa que leia dois números e mostre a soma entre eles.

In [5]:
num1 = int(input("Digite um número: "))
num2 = int(input("Digite outro: "))
soma = num1 + num2
print(f"A soma entre {num1} e {num2} é {soma}.")

Digite um número: 3
Digite outro: 4
A soma entre 3 e 4 é 7


### Exercício 004: Faça um programa que leia algo pelo teclado e mostre na tela o seu tipo primitivo e todas as informações possíveis sobre ele.

In [9]:
algumaCoisa = input("Digite algo: ")
print(f"O tipo primitivo é {type(algumaCoisa)}")

Digite algo: 5 < 8
O tipo primitivo é <class 'str'>


## Aula 7

### Exercício 005: Faça um programa que leia um número Inteiro e mostre na tela o seu sucessor e seu antecessor.

In [11]:
numero = int(input("Digite um número: "))
print(f"Número = {numero}\nSucesssor = {numero + 1}\nAntecessor = {numero - 1}")

Digite um número: 5
Número = 5
Sucesssor = 6
Antecessor = 4


### Exercício 006: Crie um algoritmo que leia um número e mostre o seu dobro, triplo e raiz quadrada.

In [17]:
from math import sqrt
numero = int(input("Digite um número: "))
print(f"Número = {numero}\nDobro = {numero * 2}\nTriplo = {numero * 3}\nRaiz Quadrada = {sqrt(numero):.2f}")'2'

Digite um número: 5
Número = 5
Dobro = 10
Triplo = 15
Raiz Quadrada = 2.24


### Exercício 007: Desenvolva um programa que leia as duas notas de um aluno, calcule e mostre a sua média.

In [18]:
nota1 = float(input("Digite uma nota: "))
nota2 = float(input("Digite outra nota: "))
media = (nota1 + nota2)/ 2
print(f"Somando as notas {nota1} e {nota2}, dá uma média de {media:.2f}.")

Digite uma nota: 6.89
Digite outra nota: 4
Somando as notas 6.89 e 4.0, dá uma média de 5.45


### Exercício 008: Escreva um programa que leia um valor em metros e o exiba convertido em centímetros e milímetros.

In [22]:
valorEmMetros = float(input("Digite o valor:(em M) "))
print(f"{valorEmMetros} equivale a {valorEmMetros * 100}cm, e {valorEmMetros * 1000}mm")

Digite o valor:(em M) 60.5
60.5 equivale a 6050.0cm, e 60500.0mm


### Exercício 009: Faça um programa que leia um número Inteiro qualquer e mostre na tela a sua tabuada.

In [25]:
numero = int(input("Digite um número: "))
for i in range(10):
  print(f"{numero} + {i + 1}  = {numero + i + 1}")

Digite um número: 3
3 + 1  = 4
3 + 2  = 5
3 + 3  = 6
3 + 4  = 7
3 + 5  = 8
3 + 6  = 9
3 + 7  = 10
3 + 8  = 11
3 + 9  = 12
3 + 10  = 13


### Exercício 010: Crie um programa que leia quanto dinheiro uma pessoa tem na carteira e mostre quantos dólares ela pode comprar.

### Exercício 011: Faça um programa que leia a largura e a altura de uma parede em metros, calcule a sua área e a quantidade de tinta necessária para pintá-la, sabendo que cada litro de tinta pinta uma área de 2 metros quadrados.

### Exercício 012: Faça um algoritmo que leia o preço de um produto e mostre seu novo preço, com 5% de desconto.

### Exercício 013: Faça um algoritmo que leia o salário de um funcionário e mostre seu novo salário, com 15% de aumento.

### Exercício 014: Escreva um programa que converta uma temperatura digitando em graus Celsius e converta para graus Fahrenheit.

### Exercício 015: Escreva um programa que pergunte a quantidade de Km percorridos por um carro alugado e a quantidade de dias pelos quais ele foi alugado. Calcule o preço a pagar, sabendo que o carro custa R$60 por dia e R$0,15 por Km rodado.

## Aula 8

### Exercício 016: Crie um programa que leia um número Real qualquer pelo teclado e mostre na tela a sua porção Inteira.

### Exercício 017: Faça um programa que leia o comprimento do cateto oposto e do cateto adjacente de um triângulo retângulo. Calcule e mostre o comprimento da hipotenusa.

### Exercício 018: Faça um programa que leia um ângulo qualquer e mostre na tela o valor do seno, cosseno e tangente desse ângulo.

### Exercício 019: Um professor quer sortear um dos seus quatro alunos para apagar o quadro. Faça um programa que ajude ele, lendo o nome dos alunos e escrevendo na tela o nome do escolhido.

### Exercício 020: O mesmo professor do desafio 019 quer sortear a ordem de apresentação de trabalhos dos alunos. Faça um programa que leia o nome dos quatro alunos e mostre a ordem sorteada.

### Exercício 021: Faça um programa em que abra e reproduza o áudio de um arquivo MP3.

## Aula 9

### Exercício 022: Crie um programa que leia o nome completo de uma pessoa e mostre:

### Exercício 023: Faça um programa que leia um número de 0 a 9999 e mostre na tela cada um dos dígitos separados.

### Exercício 024: Crie um programa que leia o nome de uma cidade diga se ela começa ou não com o nome "SANTO".

### Exercício 025: Crie um programa que leia o nome de uma pessoa e diga se ela tem "SILVA" no nome.

### Exercício 026: Faça um programa que leia uma frase pelo teclado e mostre quantas vezes aparece a letra "A", em que posição ela aparece a primeira vez e em que posição ela aparece a última vez.

### Exercício 027: Faça um programa que leia o nome completo de uma pessoa, mostrando em seguida o primeiro e o último nome separadamente.

## Aula 10

### Exercício 028: Escreva um programa que faça o computador "pensar" em um número inteiro entre 0 e 5 e peça para o usuário tentar descobrir qual foi o número escolhido pelo computador. O programa deverá escrever na tela se o usuário venceu ou perdeu.

### Exercício 029: Escreva um programa que leia a velocidade de um carro. Se ele ultrapassar 80Km/h, mostre uma mensagem dizendo que ele foi multado. A multa vai custar R$7,00 por cada Km acima do limite.

### Exercício 030: Crie um programa que leia um número inteiro e mostre na tela se ele é PAR ou ÍMPAR.

### Exercício 031: Desenvolva um programa que pergunte a distância de uma viagem em Km. Calcule o preço da passagem, cobrando R$0,50 por Km para viagens de até 200Km e R$0,45 parta viagens mais longas.

### Exercício 032: Faça um programa que leia um ano qualquer e mostre se ele é bissexto.

### Exercício 033: Faça um programa que leia três números e mostre qual é o maior e qual é o menor.

### Exercício 034: Escreva um programa que pergunte o salário de um funcionário e calcule o valor do seu aumento. Para salários superiores a R$1250,00, calcule um aumento de 10%. Para os inferiores ou iguais, o aumento é de 15%.

### Exercício 035: Desenvolva um programa que leia o comprimento de três retas e diga ao usuário se elas podem ou não formar um triângulo.